core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -103,7 +103,7 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
       val taskSortColumn = Option(parameterTaskSortColumn).map { sortColumn =>
         UIUtils.decodeURLParameter(sortColumn)
       }.getOrElse("Index")
-      val taskSortDesc = Option(parameterTaskSortDesc).map(_.toBoolean).getOrElse(false)
+      val taskSortDesc = Option(parameterTaskSortDesc).exists(_.toBoolean)
```

Comment:
If my opinion can be helpful, personally I prefer the previous one to explicitly show the default. Actually, I intendedly use this pattern in several places.